app/services/katello/applicability/applicable_content_helper.rb

```diff
@@ -0,0 +1,81 @@
+module Katello
+  module Applicability
+    class ApplicableContentHelper
+      attr_accessor :content_facet, :content_unit_class
+      
+      def initialize(content_unit_class, content_facet = nil)
+        self.content_facet = content_facet
+        self.content_unit_class = content_unit_class
+      end
+
+      def calculate_and_import
+        to_add, to_remove = applicable_differences
+        ActiveRecord::Base.transaction do
+          insert(to_add) unless to_add.blank?
+          remove(to_remove) unless to_remove.blank?
+        end
+
+        # FIXME: Debug return
+        return [to_add, to_remove]
+      end
+
+      def content_ids
+        if self.content_unit_class == ::Katello::Erratum
+          # Query for all Errata ids that are attached to the host's applicable packages
+          return ::Katello::Erratum.where(id: ::Katello::ErratumPackage.
+                                          where(nvrea: self.content_facet.applicable_rpms.pluck(:nvra)).
```

Comment:
Is this comparison between `nvrea` and `nvra` okay? It seem like internally they end up being the same.  Perhaps I should compare `filename` instead?